client/landing/stepper/declarative-flow/helpers/test/index.ts

```diff
@@ -0,0 +1,29 @@
+import { shouldRedirectToSiteMigration } from '..';
+import { STEPS } from '../../internals/steps';
+describe( 'DeclarativeFlowHelpers', () => {
+	it( 'returns true when current step, platform and origin as set properly', () => {
+		expect(
+			shouldRedirectToSiteMigration(
+				STEPS.IMPORT_LIST.slug,
+				'wordpress',
+				'site-migration-identify'
+			)
+		).toBe( true );
+	} );
+
+	it( 'returns false when current step is not importLimit', () => {
```

Comment:
Shouldn't this be `importList`?
```suggestion
	it( 'returns false when current step is not importList', () => {
```